src/emulator/gui/src/gui.cpp

```diff
@@ -153,12 +153,12 @@ static void init_background(State &gui, const std::string &image_path) {
     stbi_uc *data = stbi_load(image_path.c_str(), &width, &height, nullptr, STBI_rgb_alpha);
 
     if (!data) {
-        LOG_INFO("Could not load image from {}.", image_path);
+        LOG_ERROR("Image is Invalide format or corrupted: {}.", image_path);
```

Comment:
```suggestion
        LOG_ERROR("Image has invalid format or corrupted: {}.", image_path);
````

Reply:
Or even better: "Invalid or corrupted image: {}"

Short and sweet.